### PR TITLE
Unify manifest mode articles and fix links

### LINKED ISSUE #104: There is no description on how to change the mode vcpkg runs in.
I am a vcpkg noob, so I followed the install instructions VS 2022 linked to here: https://vcpkg.io/en/getting-started.html
 I installed a package in classic mode, and then when I restarted the console, vcpkg switched to manifest mode, and it would be nice to have docs that tell you how to switch modes.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 9263b09a-d6a7-229d-8e9d-9b8b1fae7c18
* Version Independent ID: 9263b09a-d6a7-229d-8e9d-9b8b1fae7c18
* Content: [Classic mode](https://learn.microsoft.com/en-us/vcpkg/users/classic-mode)
* Content Source: [vcpkg/users/classic-mode.md](https://github.com/Microsoft/vcpkg-docs/blob/main/vcpkg/users/classic-mode.md)
* Product: **vcpkg**
* GitHub Login: @ras0219-msft
* Microsoft Alias: **roschuma**

### LINKED ISSUE #200: Two manifest articles under Concepts
There are two manifest articles under Concepts currently. These should be reconciled or one should be moved to a different section of the docs.

### PR DESCRIPTION
**Pull request by vicroms**
Fixes #200 
Fixes #104

### CODE REVIEW

File: vcpkg/users/manifests.md [hunk 1]
```diff
@@ -1,140 +0,0 @@
...
-  "version": "0.15.2",
-  "dependencies": [
-    "boost-system",
-    "libxml2",
-    "yajl"
-  ]
-}
-```
-
-## Features
```
Comment (vicroms) [inline vcpkg/users/manifests.md:32]: @JavierMatosD 
The sections talking about features should be moved into the article you're working on. 
Most of the links in other articles should also be pointed to that future article too.

File: vcpkg/concepts/manifest-mode.md [hunk 2]
```diff
@@ -54,10 +57,12 @@ resolving version constraints, etc.
...
 
-## Manifest files in your projects
+## Manifest files in projects
+
 The main purpose of using a manifest file in your project is to declare your
-dependencies. When used on a project, you're also able to specify version 
-constraints and overrides to lock specific versions of your dependencies.
+dependencies. When using a project manifest, you're able to specify version
+constraints and overrides to lock specific versions of your dependencies; which
+is a feature not available in classic mode.
```
Comment (AugP) [inline vcpkg/concepts/manifest-mode.md:65]:
```suggestion
constraints and overrides to lock specific versions of your dependencies. This 
feature is not available in classic mode.
```

File: vcpkg/concepts/manifest-mode.md [hunk 3]
```diff
@@ -99,10 +105,82 @@ vcpkg can be configured through a `vcpkg-configuration.json` file to add more
...
+Packages installed in manifest mode will not be installed in the global
+[`installed`](../reference/installation-tree-layout.md) directory as they do in
+classic mode. Instead, each manifest gets its own installation directory named
+`vcpkg_installed`; the `vcpkg_installed` directory is created in the same
+directory that contains the manifest file.
+
+Having independent installation trees per manifest allows separation of
+dependencies between different projects. This circumvents a crucial limitation
+of classic mode, which only allows one version of each port to be installed.
+While manifest mode keeps versions of ports separated per project.
```
Comment (AugP) [inline vcpkg/concepts/manifest-mode.md:125]:
```suggestion
Manifest mode keeps versions of ports separated per project.
```

File: vcpkg/concepts/manifest-mode.md [hunk 4]
```diff
@@ -99,10 +105,82 @@ vcpkg can be configured through a `vcpkg-configuration.json` file to add more
...
+
+## Using features in project manifests
+
+Manifest files can define additive sets of functionality, behavior, and
+dependencies; through the use of
```
Comment (AugP) [inline vcpkg/concepts/manifest-mode.md:130]:
```suggestion
Manifest files can define additive sets of functionality, behavior, and
dependencies through the use of
```

File: vcpkg/concepts/manifest-mode.md [hunk 5]
```diff
@@ -11,26 +11,29 @@ ms.topic: concept-article
 
 # What is manifest mode?
 
-vcpkg has two operation modes: classic mode and manifest mode. This article
-describes the capabilities of manifest mode, which is the recommended
-mode for most users.
+vcpkg has two operation modes: [classic mode](classic-mode.md) and manifest
+mode. For most users, we recommned manifest mode.
```
Comment (aso-at-live2d-com) [inline vcpkg/concepts/manifest-mode.md:15]: > recommned

recommend?

**Pull request merged**